features/proxy_config.feature

```diff
@@ -26,3 +26,56 @@ Feature: Attach command when proxy is configured
            | xenial  |
            | bionic  |
            | focal   |
+
+    @series.lts
+    Scenario Outline: Apt proxy is configured before attach
```

Comment:
I think we can merge this test with the one for the web proxies. Since the proxy tests are creating two distinct instance, I think we can merge them to save on performance.

Reply:
Good point, I'll try that out!